### PR TITLE
EZP-32170: Fixed policy limitation custom dropdown

### PR DESCRIPTION
**Pull request by lucasOsti**
| Question      | Answer
| ------------- | ---
| Tickets       | https://issues.ibexa.co/browse/EZP-32170
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/js/scripts/core/custom.dropdown.js [hunk 1]
```diff
@@ -154,7 +160,7 @@
         }
 
         onOptionClick({ target }) {
-            const option = target.classList.contains(CLASS_CUSTOM_DROPDOWN_ITEM) ? target : target.closest(SELECTOR_ITEM);
+            const option = target.closest(`.${CLASS_CUSTOM_DROPDOWN_ITEM}`);
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/core/custom.dropdown.js:163]: You can use `SELECTOR_ITEM`

**Review by dew326 — CHANGES_REQUESTED** [review]

**Comment by dew326** [comment]
This should be merged to 3.2.

**Review by dew326 — APPROVED** [review]

**Review by GrabowskiM — APPROVED** [review]

**Review by micszo — APPROVED** [review]
Works fine now. 👍 
Tested on Ibexa DXP v3.2.1 with diff.

**Comment by lserwatka** [comment]
Could you merge it up?

**Pull request merged**